02_gateway_middleware/README.md

```diff
@@ -0,0 +1,37 @@
+# Going further
```

Comment:
We should wrap both into one tutorial. We have 01 already described in https://anbox-cloud.io/docs/usage/usecase-streaming-sdk so not necessary to repeat. Lets put a single tutorial

Reply:
I feel like repeating here isn't necessarily a bad thing as we can't guarantee readers will have gone through the docs. Starting with something simple makes the learning curve a little smoother

Reply:
Lets try to make it a single tutorial. I don't think adding the middleware widens the scope that much or makes things more complicated.

Reply:
The middleware itself not really would be fine, but I feel like having a custom connector might make things a bit complex for a first step